package/src/prmon.cpp

```diff
@@ -8,15 +8,17 @@
 #include "rapidjson/stringbuffer.h"
 #include "rapidjson/filewritestream.h"
 #include <math.h>
+#include <unistd.h>
 
 using namespace rapidjson;
 
-int ReadProcs(pid_t mother_pid, unsigned long values[4], unsigned long long valuesIO[4], bool verbose){
+int ReadProcs(pid_t mother_pid, unsigned long values[4], unsigned long long valuesIO[4], unsigned long long valuesCPU[4], bool verbose){
```

Comment:
Small point (and not your error), but `mother_pid` and `verbose` could be `const`.

Reply:
Done.